### ISSUE TITLE
[Bug] Visual artifact during editing with Elementor

### ISSUE THREAD

**Issue by XVII** [10up/windows-azure-storage#60]
There's an incompatibility with Elementor and Azure Storage for Wordpress.
The Azure Add Media image from the button overlays the textbox, preventing it from being focused.

![image](https://user-images.githubusercontent.com/446329/39300298-8bc8431a-498e-11e8-8d6e-c074d9c8e38d.png)

The button doesn't function either -- fixing the image width and height to 18px like its sibling (Add Media button) clears the textbox, however it still doesn't do anything.

![image](https://user-images.githubusercontent.com/446329/39300389-d94f4ffc-498e-11e8-882e-23155b6ff45f.png)
![image](https://user-images.githubusercontent.com/446329/39300525-5bd45c4c-498f-11e8-91a3-0ce79b558fd5.png)

It should either be hidden, or fixed.

**Comment by allan23** [comment]
@ShadowXVII Thanks for the information. 

Have you determined that the Azure plugin works fine without Elementor activated? 
Have you contacted the Elementor plugin developers about this issue?

**Comment by ShadowXVII** [comment]
Yes, the editor screen doesn't exist without the elementor plugin.

No, I haven't contacted them as I'm unsure they wouldn't make adjustments to account for particular plugins (vs plugins themselves providing the compatibility).

Px specific fixes are of course not ideal, but thought you'd have more idea of the best way to implement a fix, plugin side.

**Comment by ShadowXVII** [comment]
This is still an issue, I've created a linked issue above so the respective authors can find a resolution: pojome/elementor#5282
